.github/workflows/build-and-run-model.yaml

```diff
@@ -13,6 +13,16 @@ on:
   pull_request:
     types: [opened, reopened, synchronize, closed]
   workflow_dispatch:
+    inputs:
+      enable_upload:
+        type: boolean
+        description: Upload results to S3
+        default: false
+        required: true
+      run_note:
+        type: string
+        description: Note to include with run
+        required: true
```

Comment:
**[Nitpick, non-blocking]** Given that we intend to fall back to the value set in `params.yaml`, do we really need `run_note` to always be set by the caller? (Note that I'm not totally confident how the workflow would execute if a `run_note` is not required and a user declined to pass one in; in that case I _think_ that `inputs.run_note` would be an empty string, which might need some preprocessing in order for the fallback logic to work.)

Reply:
In the latter case (env var is empty string) then `Sys.getenv()` will take the fallback value from `params.yaml`. I want `run_note` required because otherwise I think people are unlikely to tag the purpose of their run.